public/form/login.html

```diff
@@ -53,6 +83,8 @@
 			</div>
 		</form>
 	</section>
+	<script src="js/jquery-3.1.0.min.js"></script>
+	<script src="js/function.js"></script>
```

Comment:
We probably don't need to have external JS scripts. Maybe we could just inline them as they are small. What do you think?